src/sentry/notifications/utils/participants.py

```diff
@@ -584,11 +584,12 @@ def get_recipients_by_provider(
 
     # First evaluate the teams.
     setting_type = NotificationSettingEnum(NOTIFICATION_SETTING_TYPES[notification_type])
-    controller = None
     teams_by_provider: Mapping[ExternalProviders, Iterable[RpcActor]] = {}
+
     if should_use_notifications_v2(project.organization):
+        # get by team
```

Comment:
Since this gets combined below, can we merge the two user/team calls?

Reply:
@snigdhas not easily, I thought we could but not easily. I think it's OK to make two sets of queries here, doesn't slow it down too much